### DISCUSSION TITLE
Meeting minutes of Engineering calls

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-core#9867] [General]
We (The Unity Foundation and the Oracle Labs team) meet every Tuesday and Thursday. In the past, we have kept meeting minutes and published them on the wiki page. [Example](https://github.com/micronaut-projects/micronaut-core/wiki/ZZ-20210112-Engineering-Meeting)

I would like to know the community's opinion and the @micronaut-projects/core-developers team's opinion. 

A) Should we create meeting minutes?
B) Should they be public?

Whether we resume publishing meeting minutes or not, I would like to move the old meeting minutes into their repository to have a cleaner wiki. Does anyone oppose moving the meeting minutes outside the wiki?

**Comment by jeffscottbrown** [comment]
> I would like to know the community's opinion and the https://github.com/orgs/micronaut-projects/teams/core-developers team's opinion.

Thank you for soliciting input!


> Should we create meeting minutes?

I think so.


> Should they be public?

In general, I think so.  There could be topics flagged for exception (ex. discussion about an event or something the foundation is preparing to announce to the community but aren't ready yet) but I expect most of the topics discussed in the engineering meeting would be helpful to share.


> Does anyone oppose moving the meeting minutes outside the wiki?

I do not.

**Comment by dstepanov** [comment]
I think there should be a monthly call to plan and reevaluate things that have been done to infrom the public. We should also talk about the feature major version features that might come and when.

**Reply by sdelamo** [reply]
I have created a project board to capture top level roadmap features https://github.com/orgs/micronaut-projects/projects/69

**Comment by jeffscottbrown** [comment]
> We should also talk about the feature major version features that might come and when.

Would that be new for the team or do you already do that?

**Reply by wetted** [reply]
We used to do that when we were still at OCI, as well as release retrospectives. @JasonTypesCodes was managing/driving that at the time, but that's some years ago. I left the team and returned since then.

**Reply by jeffscottbrown** [reply]
> We used to do that when we were still at OCI, as well as release retrospectives.

That is good info.  I think that should be picked back up.

**Comment by wetted** [comment]
I like the idea of capturing meeting minutes. In part because sometimes I hear things without fully absorbing them, and come to realize later the information is useful for solving something I'm working on. Meeting minutes would help in that regard, if nothing more than to tell me who to reach out to for help.

**Comment by yawkat** [comment]
I have nothing against meeting minutes (as long as I don't have to write them) but I don't think they'd be very useful either. What we discuss is mostly short-term coordination and status updates that is of limited usefulness to non core developers.

**Reply by sdelamo** [reply]
I don't think they will be particularly useful either.

**Comment by sdelamo** [comment]
We should consider than recording, even in written form, and making something public may hinder the conversation.

**Comment by sgammon** [comment]
I'd love to sit in on these every now and then, just as an observer, if you guys would allow that kind of thing 😄
